<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managementCategoryUi</class>
 <widget class="QWidget" name="managementCategoryUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupSelect">
     <property name="title">
      <string>Select Category:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="3">
       <widget class="simpleTable" name="tableCategory">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelFrom">
        <property name="text">
         <string>From</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDateEdit" name="dateFrom">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2010</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelTo">
        <property name="text">
         <string>To</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDateEdit" name="dateTo">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2010</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="buttonOk">
        <property name="text">
         <string>Ok</string>
        </property>
        <property name="icon">
         <iconset resource="../../../core/res.qrc">
          <normaloff>:/conferma24x24.png</normaloff>:/conferma24x24.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="3">
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTableView" name="tableViewDetailed">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="gridStyle">
      <enum>Qt::DashDotLine</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Detailed:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>simpleTable</class>
   <extends>QTableView</extends>
   <header location="global">simpletable.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../core/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>managementCategoryUi</receiver>
   <slot>okPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>okPressed()</slot>
  <slot>allPressed(int)</slot>
 </slots>
</ui>
